projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts

```diff
@@ -669,6 +669,10 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
             spreadInModule: true,
         },
     },
+    {
+        from: {name: 'TuiHostedDropdownComponent', moduleSpecifier: '@taiga-ui/core'},
+        to: {name: 'TuiDropdownDirective', moduleSpecifier: '@taiga-ui/core'},
```

Comment:
Shouldn't it be DropdownOpen?

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/8336/commits/f18814b2a6abc7fef10e2574806b60f8368d5def